test/manifest.js

```diff
@@ -174,19 +174,19 @@ test('uses correct base path for each file', async t => {
 	stream.write(createFile({
 		cwd: 'app/',
 		base: 'app/',
-		path: path.join('app', 'foo', 'scriptfoo-d41d8cd98f.js'),
+		path: path.join('app', 'foo', 'scriptfoo-d41d8cd98f.js').replace(/\\/g, '/'),
```

Comment:
This indeed seems to be due to a bug in the `relPath()` function. (Which is also related to #220.) With the following modification to `relPath()` the two `.replace(/\\/g, '/')` calls in this particular test aren't needed anymore:
```diff
function relPath(base, filePath) {
+	filePath = filePath.replace(/\\/g, '/');
+	base = base.replace(/\\/g, '/');

	if (filePath.indexOf(base) !== 0) {
-		return filePath.replace(/\\/g, '/');
+		return filePath;
	}

-	const newPath = filePath.slice(base.length).replace(/\\/g, '/');
+	const newPath = filePath.slice(base.length);

	if (newPath[0] === '/') {
		return newPath.slice(1);
	}

	return newPath;
}
```
I.e. in the original code the first if-test would always resolve to `true` on Windows because `file.path` was platform dependent due to `path.join()` while `file.base` was the platform independent `app/`. Which in turn resulted in the (normalized) full `file.path` to be returned instead of the relative path.